cohort_3/curriculum/workshops/portfolio/README.md

```diff
@@ -0,0 +1,271 @@
+# Portfolio
+
+## Cloud9
+
+- Go to https://c9.io, click the `Sign Up` button, and create an account
+- Click `Go to your dashboard`
+- Creating a workspace
+    - Click `Create a new workspace`
+    - Give it the name `hack-camp`
+    - Make sure the `Custom` type is selected
```

Comment:
This is selected by default for me. Is it necessary to tell them to check for something that is default? If so, we should also have them check that their workspace is public.